pkg/autoupdate/tuf/util_darwin.go

```diff
@@ -0,0 +1,21 @@
+//go:build darwin
```

Comment:
I don't think I've really reasoned about it, but I got the other way -- I'd rather split this into platform specific helpers, and not have a switch at runtime.

Reply:
My reasoning behind it is that it's simply less files to maintain. I get annoyed when I update a MacOS specific file and then CI dings me because I did not make the corresponding update on Windows. However, I would not hold up the PR for this, so I'm down with whatever =)